spring-5-webflux/pom.xml

```diff
@@ -90,12 +90,22 @@
                 <groupId>org.springframework.boot</groupId>
                 <artifactId>spring-boot-maven-plugin</artifactId>
             </plugin>
```

Comment:
Can you try removing this also?

Reply:
Thanks done, also did for the other one.